### ISSUE TITLE
Vuforia Studio Documentation 1.0.1

### ISSUE THREAD

**Issue by patrickscheper** [ptc-ar-sharing/vuforiastudio#1]
Currently we're working on creating more examples and extending the documentation in the [wiki](https://github.com/ptc-ar-sharing/vuforiastudio/wiki).

We hope to extend the documentation by adding the Javascript properties, services, events to the 2D widgets. @Unfaehig is current working on documenting the properties. 

We're also adding a shader example created by @stefanTincescu which hopefully will help users better understand the features behind the mysterious [TML widget](https://github.com/ptc-ar-sharing/vuforiastudio/wiki/TML-Text).

And as last we're going to make it easier to download examples without the need to download the entire repository or clone it via a Git client. 

## tl;dr
- Extension of the 2D widget documentation
- Shader Example Project
- Easier Project Downloads
